src/block-utils.js

```diff
@@ -1,10 +1,12 @@
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable no-undef */
 const { Scalar } = require('ffjavascript');
 
 const constants = require('./constants');
 const {
     keyBlockHeaderParams, keyTxLogs, keyTxStatus, keyTxHash, keyTxCumulativeGasUsed, keyTxEffectivePercentage,
 } = require('./block-keys-utils');
-
+const { linearPoseidon } = require('./smt-utils');
 /**
```

Comment:
I would add an empty blank line netween package import and comments of the next function